css/large.css

```diff
@@ -0,0 +1,18 @@
+#hang {
+  position: absolute;
```

Comment:
`SUGGESTION`

Since you using bootstrap, you can use the `position-absolute` Bootstrap class name for this selector element and in all other occurrences. This way the `Bootstrap framework` is fully utilized.